tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode/CspSubarrayLeafNode.py

```diff
@@ -80,7 +87,7 @@ def commandCallback(self, event):
     # Device Properties
     # -----------------
     CspSubarrayNodeFQDN = device_property(
-        dtype='str', default_value= CONST.PROP_DEF_VAL_CSP_MID_SA1
+        dtype='str', default_value="mid_csp/elt/subarray01"
```

Comment:
Can be replaced by the CONST.<<somestring>>

Reply:
ReadTheDocs is not working when CONST file variable is used. Therefore kept it as a string for now.

Reply:
It has already been tried replacing CONST.<>. But then documentation was not generating. So it is changed like that.

Reply:
But in future it will be changed to CONST.<>.